### DISCUSSION TITLE
Arctic Code Vault badge disappeared

### DISCUSSION THREAD

**Discussion by pmeredit** [community/community#19284] [Other Feature Feedback, Questions, & Ideas]
I had the artic code badge since it came out, and since the new achievements were added it seems to be gone. I definitely have commits in repos in the arctic code vault (numerous repos in the mongodb org).

Not too worried about it, mostly just curious.

**Comment by melvincwng** [comment]
@pmeredit, try raising a ticket to Github Support and let them check out why your Artic Code Vault badge has disappeared.

They usually respond within a few hours to couple of working days, depending on number of tickers received per day.

I have linked the [Github support page](https://support.github.com/contact/account) here for your convenience. 

Hope this helps!

**Comment by radar-blip** [comment]
You could try turning off 'show achievements on profile' and turn back on again. Under settings.

**Comment by doganarif** [comment]
It had happened to me when I remove my old email from my Github account. Maybe checking your commit ownership would be good.
